app/models/spree/package_builder.rb

```diff
@@ -106,7 +106,7 @@ def packages(package)
       end
 
       item_specific_packages.each do |package|
-        packages << ::ActiveShipping::Package.new(package.at(0), [package.at(1), package.at(2), package.at(3)], units: :imperial)
+        packages << ::ActiveShipping::Package.new(package.weight * multiplier, [package.length, package.width, package.height], units: units)
```

Comment:
:+1: for removing the hard-coded imperial units. The default value for ActiveShipping is metric if I am not mistaken. Does this mean we need to initialize `Spree::ActiveShipping::Config[:units]` to `:imperial` by default to preserve the original functionality?